components/File/src/index.scss

```diff
@@ -0,0 +1,59 @@
+.denhaag-file {
+  display: flex;
+  color: inherit;
+  text-decoration: inherit;
+  border: var(--denhaag-file-border);
+
+  &--focus,
+  &:focus {
+    outline: var(--denhaag-file-focus-outline);
```

Comment:
For outline we also don't use the shorthand but the separate constituent properties: https://developer.mozilla.org/en-US/docs/Web/CSS/outline#constituent_properties